index.d.ts

```diff
@@ -0,0 +1,10 @@
+declare module 'coordinate-parser' {
+    class CoordinateParser {
```

Comment:
This module does not provide any class `CoordinateParser`. Is this line correct?

Reply:
Compiled JS code corresponds to a typescript class. I change name to Coordinates to match JS code. Thank you pointing this out.

Reply:
Thanks, I wasn't sure if the name here matters to TypeScript.